changhyun/app.js

```diff
@@ -33,4 +33,18 @@ app.get("/ping", function (req, res, next) {
   return res.status(200).json({ message : "pong" });
 })
 
+app.post('/create', async (req, res) => {
```

Comment:
```suggestion
app.post('/users', async (req, res) => {
```
창현님! 현재 보시면 app.post() 즉 http method의 여러 종류 중 post method를 활용해 client에서 보내는 request를 처리 하는 api를 작성해주셨습니다.

http post method에는 method 이름 자체로도 '무언가를 등록' 한다는 의미가 전달되기 때문에 `/create` 라고 엔드포인트를 작성해주신것을 더해 생각해 본다면 등록, 생성 한다는 의미가 중복으로 적용되게 됩니다. 그렇기에 post 메소드로 `/users` 라는 엔드포인트를 가진 api 라면 http 메소드와 엔드포인트를 종합하여 읽어보았을 때 users를 등록 한다는 의미를 쉽게 전달할 수 있습니다.

이렇듯 자명한(그 자체만으로도 설명이 되는) 엔드포인트를 RESTful 하게 설계된 엔드포인트이자 API라고 말하는데, 이는 차주에 RESTful API 세션을 통해 한 번 더 자세하게 짚고 배워볼 예정이니, '이런식으로 엔드포인트를 작성하면 되는구나~' 라는 느낌만 가져가시면 될 것 같아요!

수정 부탁드립니다 :)